2021/03/02 18:27:22 [error] 23#23: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9009", host: "localhost:8090"
2021/03/02 18:27:24 [error] 23#23: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.20.0.3:9009", host: "localhost:8090", referrer: "http://localhost:8090/"
2021/03/03 07:43:01 [emerg] 1#1: host not found in upstream "php-fpm" in /etc/nginx/nginx.conf:33
2021/03/03 07:51:44 [error] 23#23: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.21.0.2:9009", host: "localhost:8090"
2021/03/03 07:51:46 [error] 23#23: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.21.0.2:9009", host: "localhost:8090", referrer: "http://localhost:8090/"
2021/03/03 08:00:57 [error] 23#23: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.21.0.2:9009", host: "localhost:8090"
2021/03/03 08:00:58 [error] 23#23: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.21.0.2:9009", host: "localhost:8090", referrer: "http://localhost:8090/"
2021/03/03 08:01:18 [error] 23#23: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.21.0.2:9009", host: "localhost:8090"
2021/03/03 08:08:40 [error] 23#23: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.21.0.2:9009", host: "localhost:8090"
2021/03/03 08:08:44 [error] 23#23: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.21.0.2:9009", host: "localhost:8090", referrer: "http://localhost:8090/"
2021/03/03 08:08:50 [error] 23#23: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.21.0.2:9009", host: "0.0.0.0:8090"
2021/03/03 08:08:52 [error] 23#23: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.21.0.2:9009", host: "0.0.0.0:8090", referrer: "http://0.0.0.0:8090/"
